src/pages/vaults/VaultDepositWithdrawForm.tsx

```diff
@@ -172,10 +183,13 @@ export const VaultDepositWithdrawForm = ({
     if (isSubmitting) {
       return;
     }
+    const userOperationId = crypto?.randomUUID() ?? Date.now().toString();
```

Comment:
probably fine to just use Date.now().toString() here - seems nicer to have a consistent format in amplitude